src/main/java/com/microsoft/lst_bench/util/StringUtils.java

```diff
@@ -50,6 +50,13 @@ public static String quote(String str) {
     return "'" + str + "'";
   }
 
+  public static String replaceSingleQuotes(String str) {
```

Comment:
Maybe you could call the method `escapeSql` or anything along those lines, since we might extend it in the future to other cases.